### PR TITLE
docs: add documentation for revoking and activating a VC

### PR DESCRIPTION
**Pull request by huynguyen-hl**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR adds guidance on how to revoke and activate a VC using APIs.

## Related Tickets & Documents

https://github.com/gs-gs/fa-ag-trace/issues/595

## Mobile & Desktop Screenshots/Recordings

image
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 [vc-kit doc site](https://uncefact.github.io/vckit/)
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: documentation/docs/get-started/api-server-get-started/basic-operations.md [hunk 1]
```diff
@@ -270,3 +270,51 @@ To get the rendered VC, you can try this endpoint [`/renderCredential`](http://l
 :::tip
 The document value is a string which is a encoded HTML string. You can use any online [Base64](https://www.base64decode.org/) tool out there to decode it. Good luck!
 :::
+
+## Revoking a VC
+
+To revoke a VC, use the following VCKit API endpoint: [/agent/setBitstringStatus](http://localhost:3332/agent/setBitstringStatus).
```
Comment (ldhyen99) [inline documentation/docs/get-started/api-server-get-started/basic-operations.md:276]: Hi, will we use only the Bitstring Status List or will we also support the Revocation list? I think this line would be good if only the Bitstring Status List were available.

**Comment by namhoang1604** [comment]
Should we introduce the issuing bitstring status flow?

File: documentation/docs/get-started/api-server-get-started/basic-operations.md (hunk 1 above)
Comment (huynguyen-hl) [reply documentation/docs/get-started/api-server-get-started/basic-operations.md:276]: The `RevocationStatusList` is not compatible with VCDM v2, so we should only support the `BitstringStatusList` for now.

**Comment by huynguyen-hl** [comment]
@namhoang1604, Yes, I agree with you about adding the issuing `BitstringStatusList` flow to the documentation site. However, since there are currently no flow charts in the entire site, I believe we need a ticket to create flow charts for all the APIs that already have documentation. If we only include the flow chart for the issuing `BitstringStatusList`, it might make the documentation site a bit inconsistent.

**Review by namhoang1604 — APPROVED** [review]
LGTM

**Review by ldhyen99 — APPROVED** [review]

**Pull request merged**